src/sections/strategy.html

```diff
@@ -0,0 +1,25 @@
+<div class="lg:px-36 px-10 md:px-20 relative">
+    <div class="w-full flex justify-center sticky top-0 z-20">
+        <img src="/images/small-yellow-shape.svg" alt="" class=" lg:w-[18%]  w-[60%] md:w-[25%]">
+    </div>
+    <div class="grid lg:grid-cols-2 lg:gap-60 pt-32 grid-rows-1 gap-20 md:gap-20">
+          <div class=" text-3xl lg:text-4xl md:text-4xl font-medium  flex flex-col lg:gap-16 gap-12">
+               <p>Brand strategy</p>
+               <p>Corporate Identity &  Website development</p>
+               <p>SEO & copywriting</p>
+               <p>Media Relations</p>
+               <p>Digital marketing & content production</p>
+               <p>Influencer relations</p>
+          </div>
+          <div class="relative lg:block md:flex md:justify-end">
+            <img src="/images/brand-green-shape.svg" alt="" class=" lg:h-[95%] h-[100%] md:h-[97%] ">
+            <p class="absolute lg:top-[22%]  top-[20%] lg:w-[60%] w-[78%] xl:w-[65%] lg:left-20  left-10 text-gray-150 text-base md:w-[42%] md:left-[45%] md:top-[23%] md:text-xl xl:left-14 xl:top-[20%] xl:leading-tight">Our strategic approach is a collaborative journey that 
```

Comment:
avoid unnecessary spaces in your codebase
```suggestion
            <p class="absolute lg:top-[22%] top-[20%] lg:w-[60%] w-[78%] xl:w-[65%] lg:left-20 left-10 text-gray-150 text-base md:w-[42%] md:left-[45%] md:top-[23%] md:text-xl xl:left-14 xl:top-[20%] xl:leading-tight">Our strategic approach is a collaborative journey that 
```